style/ember/README.md

```diff
@@ -3,6 +3,11 @@ Ember
 
 * Don't put a space between the opening handlebars braces and the variable.
 * Prefer components over partials.
+* Prefer `ember-fetch` over `ic-ajax`. ([sample][ember-fetch-sample])
+  [documentation][ember-fetch]
```

Comment:
I think we'd ideally link `ember-fetch` itself. If not, maybe we should just remove it.